modules/cas_cache/layer_cache_management.c

```diff
@@ -1422,6 +1417,15 @@ int cache_mngt_remove_core_from_cache(struct kcas_remove_core *cmd)
 	if (result)
 		return result;
 
+	result = _cache_mngt_remove_core_is_active(cache, cmd->core_id);
+	if (result) {
+		printk(KERN_WARNING OCF_PREFIX_SHORT
+				"Core is already inactive. To manage the "
+				"inactive core use '--remove-inactive' "
+				"command.\n");
+		goto put;
```

Comment:
it would be more user friendly to print this message in casadm - this would require adding new return code.